arrow/arrow.py

```diff
@@ -864,13 +905,17 @@ def format(self, fmt="YYYY-MM-DD HH:mm:ssZZ", locale="en_us"):
         return formatter.DateTimeFormatter(locale).format(self._datetime, fmt)
 
     def humanize(
-        self, other=None, locale="en_us", only_distance=False, granularity="auto"
-    ):
+        self,
+        other: Optional[Union["Arrow", date]] = None,
+        locale_code: str = "en_us",
+        only_distance: bool = False,
+        granularity: Union[List[str], str] = "auto",
+    ) -> str:
         """ Returns a localized, humanized representation of a relative difference in time.
 
         :param other: (optional) an :class:`Arrow <arrow.arrow.Arrow>` or ``datetime`` object.
             Defaults to now in the current :class:`Arrow <arrow.arrow.Arrow>` object's timezone.
-        :param locale: (optional) a ``str`` specifying a locale.  Defaults to 'en_us'.
+        :param locale_code: (optional) a ``str`` specifying a locale.  Defaults to 'en_us'.
```

Comment:
mypy in this instance complained that the variable and the function parameter were named the same and that it was assigned to afterwards, so I renamed the function parameter, since it's just being used twice instead of changing the locale variable in 25 places inside of that function.

I think it also makes it clearer in naming terms that we want the locale language code, instead of something like the actual name of the locale class.